source/sass/patterns/molecules/figure.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.figure {
+  @include block-spacing($end: $baselinegrid-space-medium);
+  @include figure-caption-typography();
```

Comment:
Better on the caption? This would affect anything else in the figure otherwise (eg tables, quotes).

Reply:
Yes. There needs to be a class on `<figcaption>` which can be used to add this mixin.